### ISSUE TITLE
Narrator does not announce position number of the variables mentioned in the expression.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1388]
A team member will update this issue with more details soon.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
